aws/data_source_aws_autoscaling_group_test.go

```diff
@@ -89,7 +77,8 @@ resource "aws_autoscaling_group" "bar" {
 }
 
 data "aws_autoscaling_group" "good_match" {
-  name = aws_autoscaling_group.foo.name
+  name       = aws_autoscaling_group.foo.name
+  depends_on = [aws_autoscaling_group.foo]
```

Comment:
`depends_on` here does not work with Terraform 0.12, unfortunately.

```suggestion
  name = aws_autoscaling_group.foo.name
```

The dependency problem will resolve itself with Terraform 0.14.

Reply:
Doesn't work in 0.13. Blergh.